docs/website/docs/dlt-ecosystem/verified-sources/filesystem/basic.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Filesystem source
+description: Learn how to set up and configure
+keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+---
+import Header from '../_source-info-header.md';
+<Header/>
+
+Filesystem source is a generic source that allows loading files from remote locations (AWS S3, Google Cloud Storage, Google Drive, Azure) or the local filesystem seamlessly. Filesystem source natively supports `csv`, `parquet`, and `jsonl` files and allows customization for loading any type of structured files.
+
+To load unstructured data (`.pdf`, `.txt`, e-mail), please refer to the [unstructured data source](https://github.com/dlt-hub/verified-sources/tree/master/sources/unstructured_data).
+
+## Quick example
+
+```py
+import dlt
+from dlt.filesystem import filesystem, read_parquet
+
+filesystem_resource = filesystem(
+  bucket_url="file://Users/admin/Documents/parquet_files",
+  file_glob="**/*.parquet"
+)
+filesystem_pipe = filesystem_resource | read_parquet()
+filesystem_pipe.apply_hints(incremental=dlt.sources.incremental("modification_date"))
+
+# We load the data into the table_name table
+pipeline = dlt.pipeline(pipeline_name="my_pipeline", destination="duckdb")
+load_info = pipeline.run(filesystem_pipe.with_name("table_name"))
+print(load_info)
+print(pipeline.last_trace.last_normalize_info)
+```
+
+## Setup
+
+### Prerequisites
+
+Please make sure the `dlt` library is installed. Refer to the [installation guide](../../../getting-started).
+
+### Initialize the filesystem source
+
+To get started with your data pipeline, follow these steps:
+
+1. Enter the following command:
+
+   ```sh
+   dlt init filesystem duckdb
+   ```
+
+   [dlt init command](../../../reference/command-line-interface) will initialize
+   [the pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/filesystem_pipeline.py)
+   with the filesystem as the source and [duckdb](../../destinations/duckdb.md) as the destination.
+
+2. If you'd like to use a different destination, simply replace `duckdb` with the name of your
```

Comment:
2. If you would like to use a different destination, simply replace `duckdb` with the name of your
   preferred [destination](../../destinations).